module/VuFindSearch/src/VuFindSearch/Backend/EDS/SearchRequestModel.php

```diff
@@ -211,6 +223,41 @@ public function setParameters($parameters = [])
         }
     }
 
+    /**
+     * Set validation config
+     *
+     * @param ?Config $validationConfig Validation config
+     *
+     * @return void
+     */
+    public function setValidationConfig($validationConfig)
+    {
+        $this->validationConfig = $validationConfig;
+    }
+
+    /**
+     * Checks whether the search model is valid for the EDS API.
+     *
+     * @return bool Returns false if any model parameters are invalid.
+     */
+    public function isValid()
+    {
+        if (!($this->validationConfig->ContentProvider ?? true)) {
```

Comment:
Do you want validation to default to true, or should this default to false? (My impression is that this code contradicts the comments in the .ini file, but maybe I'm misinterpreting something).

Reply:
I think it's correct?  Validation does default to true.  The check here short-circuits the validation if it's actually set to false.

Reply:
You're right, the comments and the code match, but my real question is: do we want validation enabled by default? I thought the idea was that we were worried that it might have unwanted side effects, so we were going to leave it off by default, but encourage people to turn it on when needed for Blender purposes. But maybe I misunderstood!

Reply:
Ah yes, I missed that in your original suggestion!  Ok, I will turn it off by default.

Reply:
Done!